hack/templates/samples/mellanox.com_v1alpha1_nicclusterpolicy.template

```diff
@@ -57,35 +57,11 @@ spec:
           }
         ]
       }
-  sriovDevicePlugin:
```

Comment:
@e0ne  can we leave it just be with comment?

Reply:
unfortunately, we can't do it for OCP